### DISCUSSION TITLE
Disable the backward and forward button on locked screen

### DISCUSSION THREAD

**Discussion by jeremieoneglancestudio** [mediagrid/capacitor-native-audio#9] [2. Feature Ideas]
Hello,

Currently, it is not possible to disable the backward and forward button on the locked screen. Possible to disable it? It is particularly useful for audio stream, which is not necessary for.

Thank you for your work!

**Comment by wsamoht** [comment]
This has been completed in https://github.com/mediagrid/capacitor-native-audio/pull/15.

A `showSeekBackward` and `showSeekForward` option has been added in the `create` method.
